hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -120,6 +121,23 @@ def min_rep(locus, ref, alt):
             ['locus', 'alleles'],
             is_sorted=True))
 
+
+# FIXME Placeholder
+def densify(mt):
+    return mt
+
+
+def summarize(mt):
+    mt = densify(mt)
+    return mt.annotate_rows(info=mt.info.annotate(
+        DP=hl.agg.sum(mt.entry.DP),  # some DPs may have been missing during earlier combining operations
+        BaseQRankSum=hl.median(hl.agg.collect(mt.entry.BaseQRankSum)),
```

Comment:
Is this the best way to take the median over the entries?

Reply:
yes.

Reply:
😢

Reply:
I was wrong.